examples/resources/grafana_alert_rule/_acc_multi_rule_group.tf

```diff
@@ -0,0 +1,161 @@
+resource "grafana_folder" "compound_rule_folder" {
+    title = "My Compound Alert Rule Folder"
+}
+
+resource "grafana_alert_rule" "my_multi_alert_group" {
+    name = "My Multi-Alert Rule Group"
+    folder_uid = grafana_folder.compound_rule_folder.uid
+    interval_seconds = 240
+    org_id = 1
+    rules {
```

Comment:
Nit: should we call this `rule` instead of `rules`?

Reply:
Good idea, done